File: Examples/VisualBasic/Files/Handling/SaveFileInExcel972003format.vb
Imports System.IO

Imports Aspose.Cells

Namespace Files.Handling
    Public Class SaveFileInExcel972003format
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Creating a Workbook object
            Dim workbook As New Workbook()

            ' Your Code goes here for any workbook related operations

            ' Save in Excel 97 – 2003 format
            workbook.Save(dataDir & "output.xls")

            ' OR
            workbook.Save(dataDir & "output.xls", New XlsSaveOptions(SaveFormat.Excel97To2003))
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Files/Handling/SaveFileInExcel972003format.cs
using System.IO;

using Aspose.Cells;

namespace CSharp.Files.Handling
{
    public class SaveFileInExcel972003format
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            // Creating a Workbook object
            Workbook workbook = new Workbook();

            // Your Code goes here for any workbook related operations

            // Save in Excel 97 – 2003 format
            workbook.Save(dataDir + "output.xls");

            // OR
            workbook.Save(dataDir + "output.xls", new XlsSaveOptions(SaveFormat.Excel97To2003));
            // ExEnd:1
           }
         }
      }
